addons/im_livechat/static/src/components/composer/composer.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-inherit="mail.Composer" t-inherit-mode="extension">
+        <xpath expr="//*[@name='buttonAttachment']" position="replace">
```

Comment:
No need to add a `name` attribute when we have an explicit class such as `o_Composer_buttonAttachment` in this case.